pkg/sdkv2/resources/job_acceptance_test.go

```diff
@@ -236,6 +236,98 @@ func TestAccDbtCloudJobResourceTriggers(t *testing.T) {
 	})
 }
 
+func TestAccDbtCloudJobCISettings(t *testing.T) {
+
+	jobName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	projectName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	environmentName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: acctest_helper.TestAccProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckDbtCloudJobDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDbtCloudJobResourceCISettings(
+					jobName,
+					projectName,
+					environmentName,
+					false,
+					true,
+				),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDbtCloudJobExists("dbtcloud_job.ci_job"),
+					resource.TestCheckResourceAttr("dbtcloud_job.ci_job", "name", jobName),
+					resource.TestCheckResourceAttr("dbtcloud_job.ci_job", "run_lint", "false"),
+					resource.TestCheckResourceAttr("dbtcloud_job.ci_job", "errors_on_lint_failure", "true"),
```

Comment:
Maybe a bigger discussion for the future, but I read somewhere that we actually might not need to check attributes that are not `Computed`. The testing framework actually checks that all the non Computed values are OK based on the config (e.g. it makes sure that a `plan` after an `apply` would not raise any diff). I know that this is not the way that the provider has been written so far though.

Reply:
That makes sense! Let me read up on it and maybe I can remove those!